backend/apis/clients/client-interactions/client-interaction-utils.api.js

```diff
@@ -0,0 +1,74 @@
+const mysql = require("mysql");
+const { responseFullName } = require("../../utils/transform-utils");
+
+const modifiableInteractionTypes = [
+  "clientInteraction:created",
+  "clientInteraction:updated"
+];
+
+exports.insertClientInteractionsQuery = function(params) {
+  if (params.detailIdIsLastInsertId) {
+    return mysql.format(
+      `
+				INSERT INTO clientInteractions (clientId, title, serviceId, interactionType, description, dateOfInteraction, duraction, createdBy, detailId)
+				VALUES (?, ?, ?, ?, ?, ?, ?, ?, LAST_INSERT_ID());
+			`,
+      [
+        params.clientId,
+        params.title,
+        params.serviceId,
+        params.interactionType,
+        params.description,
+        params.dateOfInteraction,
+        params.duraction,
+        params.location,
+        params.isDeleted,
+        params.createdBy
+      ]
+    );
+  } else {
+    return mysql.format(
+      `
+				INSERT INTO clientLogs (clientId, title, serviceId, interactionType, description, dateOfInteraction, duraction, createdBy, detailId)
```

Comment:
Why is this function sometimes inserting into clientInteractions and other times inserting into clientLogs?

Reply:
Was removed as part of above request